docs/types.rst

```diff
@@ -304,6 +304,9 @@ String literals are written with either double or single-quotes (``"foo"`` or ``
 
 String literals support escape characters, such as ``\n``, ``\xNN`` and ``\uNNNN``. ``\xNN`` takes a hex value and inserts the appropriate byte, while ``\uNNNN`` takes a Unicode codepoint and inserts an UTF-8 sequence.
 
+Solidity doesn't have any inbuilt string utility functions, you first have to convert it to a ``bytes`` type before any comparisons or manipulation.
+
+You can find examples of utility functions in `stringUtils.sol <https://github.com/ethereum/dapp-bin/blob/master/library/stringUtils.sol>`_ and `solidity-stringutils <https://github.com/Arachnid/solidity-stringutils>`_.
```

Comment:
This will soon fail to compile, so I'm not sure if we should mention it.

Reply:
@chriseth OK, out of interest, what will be the future solution to string utility functions?

Reply:
Someone has to update it :)

Reply:
Actually no, it seems to be up to date! Yeah!